### PR TITLE
[Feat] - Improvements from feedback

### PR DESCRIPTION
**Pull request by mariano-aguero**
Add some improvements from feedback

### CODE REVIEW

File: server/helpers/sendTelegram.js [hunk 1]
```diff
@@ -51,24 +51,29 @@ ${welcomeText}`,
...
-        currentRound = await getLivepeerCurrentRound()
+        currentRoundObject = await getLivepeerCurrentRoundInfo()
       }
     } catch (err) {
       retry()
     }
   })
-  if (!delegatorAccount || !transcoderAccount || !currentRound) {
-    throw new Error('There is no telegram to send')
+  console.log(`Delegator account ${JSON.stringify(delegatorAccount)}`)
```
Comment (Agupane) [inline server/helpers/sendTelegram.js]: Could you delete this or remplace it with a logger?, thanks

File: server/helpers/utils.js [hunk 2]
```diff
@@ -94,6 +94,17 @@ const subscriptionSave = async data => {
 
   const { address, chatId } = data
 
+  const { getLivepeerDelegatorAccount } = require('./livepeerAPI')
+  const delegatorAccount = await getLivepeerDelegatorAccount(address)
+
+  if (delegatorAccount.status !== 'Bonded') {
```
Comment (Agupane) [inline server/helpers/utils.js]: Could you delegatorAccount && delegatorAcount.status check ?, thanks

Comment (mariano-aguero) [reply server/helpers/utils.js]: Done

File: server/helpers/sendTelegram.js (hunk 1 above)
Comment (mariano-aguero) [reply server/helpers/sendTelegram.js]: @Agupane This log is for control purposes, Papertrail will register it and we can use it to verify possible problems. Logger is not used in the backend, it is more to use with chrome dev tools. Thanks for the advice, but it is recommended to keep like this.
Regards

**Pull request merged**